packages/psss-server/src/app/addRoutesToApp.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { Express, Response, NextFunction } from 'express';
+import { InternalServerError, RespondingError } from '@tupaia/utils';
+import { PsssRequest } from '../types';
+import { LoginRouteHandler, TestHandler } from '../routes';
+
+const handleWith = (Handler: any) => (req: PsssRequest, res: Response, next: NextFunction) => {
+  const handler = new Handler(req, res, next);
+  return handler.handle();
+};
+
+const handleError = (
+  err: RespondingError | Error,
+  req: PsssRequest,
+  res: Response,
+  next: NextFunction,
```

Comment:
@rohan-bes this needed four arguments to know it should handle errors (plus see the first conditional taken from web-config-server)